src/components/dls/Switch/Switch.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+
+import styles from './Switch.module.scss';
+
+// reference: https://vercel.com/design/switch
+
+type Item = {
+  name: string;
+  value: string;
+  width: number;
+  disabled?: boolean;
+};
+export enum SwitchSize {
+  Small = 'small',
+  Normal = 'normal',
```

Comment:
I think "Normal" should be renamed to "Medium" and also inside the css class names for the sake of consistency with other components when it comes to sizing

Reply:
agree 👍

Reply:
ditto